WordPress/Classes/ViewRelated/Activity/RewindStatusRow.swift

```diff
@@ -0,0 +1,24 @@
+
+struct RewindStatusRow: ImmuTableRow {
+
+    typealias CellType = RewindStatusTableViewCell
+
+    static let cell: ImmuTableCell = {
+        let nib = UINib(nibName: "RewindStatusTableViewCell", bundle: Bundle(for: CellType.self))
+        return ImmuTableCell.nib(nib, CellType.self)
+    }()
+
+    let action: ImmuTableAction? = nil
```

Comment:
I think we can drop the assignment. Unless I missed a Swift language change, optionals are auto-initialized as `nil`

Reply:
Note that this is a `let`. It's part of the `ImmuTableRow` contract, but we don't want an action for this kind of row. Setting this as `nil` here means that it will _always_ be `nil` and won't be part of the automatically synthesised `init` :)

Reply:
Ah! I missed that detail.  Fair enough then :)